### PR TITLE
[13.0][ADD] stock_split_picking: Mode selection to split in different ways

### PR DESCRIPTION
**Pull request by JordiBForgeFlow**
Forward port of https://github.com/OCA/stock-logistics-workflow/pull/714

cc @ForgeFlow @AaronHForgeFlow

### CODE REVIEW

**Review by AaronHForgeFlow — APPROVED** [review]
Functional tests, split by:
* by done quantity
* by stock moves
* by manual selection

All of them LGTM.

File: stock_split_picking/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -2,3 +2,9 @@ This module adds a "Split" button on the outgoing pickings form.
 
 It works like the classical picking Transfer but it leaves both pickings
 (picking and its backorder) as confirmed without processing the transfer.
+<<<<<<< HEAD
+=======
+
+You can also choose to put all moves into separate pickings, or select moves
+to be put into a new picking.
+>>>>>>> e7822196... [ADD] stock_split_picking: Mode selection to split in different ways
```
Comment (LoisRForgeFlow) [inline stock_split_picking/readme/DESCRIPTION.rst]: Tested locally and looks good, yet there are some conflicts and issues with pre-commit. I'm now fixing them.

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM

**Pull request merged**